context/GlobalContext.tsx

```diff
@@ -28,7 +28,7 @@ const defaultPreferences: Preferences = {
 };
 
 interface UserData {
-  currentGroupId?: string;
+  currentGroupId: string | null | undefined;
```

Comment:
why we changed that from optional to the required but we accept null and undefiend ?